mod/thewire/views/default/js/thewire.php

```diff
@@ -14,12 +14,10 @@
 		}
 	};
 
-	$("#thewire-textarea").live({
-		input: callback,
-		onpropertychange: callback
-	});
+	$(document).on('input', "#thewire-textarea", callback);
+	$(document).on('onpropertychange', '#thewire-textarea', callback);
```

Comment:
Yea, this should just be propertychange. I could probably even register both of them in one go with:

```
$(document).on('input propertychange', '#thewire-textarea', callback);
```